### DISCUSSION TITLE
[Idea]: use native browser SVG support

### DISCUSSION THREAD

**Discussion by mtdvlpr** [sircharlo/meeting-media-manager#2928] [Ideas] [answered]
### Is your idea related to a problem?

@sircharlo, I'm wondering why we need to convert svg's to jpg when the browser supports SVG out of the box.

### Describe your idea

Couldn't we just leave svg as is and just display it?

### Describe alternatives you've considered

.

**Comment by sircharlo** [comment, marked as answer]
I had issues with the black background of the media window... Charts usually show up wonky because they have no background color defined. Converting to JPG is essentially only to add the white background that we expect.
